README.md

```diff
@@ -72,6 +72,10 @@ In `package.json`'s `"scripts"` section, add:
 
 Pre-commit mode. Under this flag only staged files will be formatted, and they will be re-staged after formatting.
 
+### `--branch`
+
+When in `staged` pre-commit mode, use this flag to compare changes with the specified branch. Defaults to `master` branch.
```

Comment:
Shouldn't this read "When _not _in `staged `"?

Reply:
Ah, yes, yes it should! I'll update.